// ng-confirm-box/esm2022/lib/ng-confirm.module.mjs
import { NgModule } from '@angular/core';
import { NgConfirmComponent } from './ng-confirm.component';
import * as i0 from "@angular/core";
class NgConfirmModule {
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule }); }
    static { this.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmModule, declarations: [NgConfirmComponent], exports: [NgConfirmComponent] }); }
    static { this.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmModule }); }
}
export { NgConfirmModule };
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        NgConfirmComponent
                    ],
                    imports: [],
                    exports: [
                        NgConfirmComponent
                    ]
                }]
        }] });
