// ng-confirm-box/esm2022/lib/ng-confirm.service.mjs
import { Injectable } from '@angular/core';
import { Subject } from 'rxjs';
import * as i0 from "@angular/core";
class NgConfirmService {
    constructor() {
        this.displayModa$ = new Subject();
    }
    showConfirm(message, yes, no) {
        this.displayModa$.next({
            isVisible: true,
            messageText: message,
            yes: () => {
                yes();
                this.closeConfirm();
            },
            no: () => {
                no();
                this.closeConfirm();
            },
        });
    }
    closeConfirm() {
        this.displayModa$.next(false);
    }
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmService, deps: [], target: i0.ɵɵFactoryTarget.Injectable }); }
    static { this.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmService, providedIn: 'root' }); }
}
export { NgConfirmService };
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.0.0", ngImport: i0, type: NgConfirmService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });
